shell/platform/tizen/BUILD.gn

```diff
@@ -161,6 +161,7 @@ template("embedder_for_profile") {
     public_deps = [ ":flutter_engine" ]
 
     deps = [
+      "//flutter/runtime:libdart",
```

Comment:
Do we need this on all runtime modes?

Reply:
It's not required for all runtime mode unless `--unoptimized` is set.

Reply:
It was not required even when `--unoptimized` is set in my case.

Reply:
I've turned it off on builds with `enable_profiling` set to `false`, to decrease the size of release builds.